### DISCUSSION TITLE
What is Stream_204?

### DISCUSSION THREAD

**Discussion by Wabagg12345** [ftde0/yt2009#191] [Q&A] [answered]
So, I'm also working with YouTube on the Wii and whenever I try to play a video it requests from s.youtube.com/stream_204. I've tried researching on what it is or was and couldn't find anything about it. So if anyone knows about this, please let me know

**Comment by aldairbotherotaku** [comment]
may be ads, since it is blocked in many hosts files

**Comment by ftde0** [comment, marked as answer]
stream_204 and player_204 are logging requests, which could explain it being blocked described above. HOWEVER /stream_204 requests could also be made in case there was an error with video playback.

in case of wii's apiplayer, a `logPlayerEvent` function is used to initiate sending such requests, and if you're looking for the source of `stream_204` requests, they are made whenever logData is `streamingerror`, which should sound pretty self-explanatory.

however, as `streamingerror` itself in an error request can be pretty vague, you CAN text search for `streamingerror` within the apiplayer swf and find `logPlayerEvent`s that mention `streamingerror`.

an example string you may find:
```
this.logPlayerEvent("streamingerror",_loc2_);
```

there will be several matches like this, all of which you can look at the function name and add that to the `streamingerror`, as that will be sent with the request (and thus you will be able to see in the network tab).

for example, such `logPlayerEvent` call will be in a function called `onFormatUnset`, which could happen when no formats are sent OR the formats can't be played on your setup (for example h264 mp4's will have trouble playing on flash 8 and they will send a `formatunset` because of that).

you can change the `logPlayerEvent` call in `onFormatUnset` to better reflect what the streaming error is about.

```
this.logPlayerEvent("streamingerror",_loc3_);
```

could become

```
this.logPlayerEvent("streamingerror_formatunset",_loc3_);
```

while this could now send a /player_204 instead of /stream_204 (`streamingerror` is hardcoded as a thing to be sent to `/stream_204`), it could give you a better insight as to what's going on within the player.

**Comment by Wabagg12345** [comment]
Ok, Thank you for responding, this helps me out a lot

**Comment by Wabagg12345** [comment]
Ok, so I figured all that out and I did do the thing where you can make it download a flv and....
![IMG_4039](https://github.com/ftde0/yt2009/assets/165488542/bf329527-5e1b-4f33-bf1a-80ecafb68964)
If you need more context, I'll provide more info

**Reply by SavunOski** [reply]
I got the same exact error using Dolphin emulator on my instance as well

**Comment by FurblandChannel** [comment]
This might be a problem with the instance you’re using. If you have access to the instance's logs I suggest checking them and seeing if you notice anything off.
